Monika After Story/game/script-greetings.rpy

```diff
@@ -2615,20 +2615,83 @@ label greeting_back_from_work:
         m 1hua "Oh, welcome back, [player]!"
         menu:
             m "Did you have a good day at work today?"
+            "I had a {i}really{/i} good day today.":
+                m 1sub "That's {i}amazing{/i}, [player]!"
+                m 1hub "I'm really happy that you had such a great day today!"
+                m 1eua "I can only imagine how well you work on a good day."
+                m 1hua "Maybe you'll even move up a bit soon!"
+                m 1eua "Anyway, welcome home."
+                m 3tsbsa "Would you like your dinner, your bath, or..."
```

Comment:
Probably make this lower chance random and only if we've seen the corresponding greet before